src/waveform/guitick.h

```diff
@@ -3,23 +3,26 @@
 
 #include <QObject>
 
+#include "control/controlobject.h"
 #include "util/duration.h"
+#include "util/memory.h"
 #include "util/performancetimer.h"
 
-class ControlObject;
-
+// A helper class that manages the "guiTickTime" COs, that drive updates of the
+// GUI from the VsyncThread at the user's configured FPS (possibly downsampled).
 class GuiTick : public QObject {
     Q_OBJECT
   public:
     GuiTick(QObject* pParent = NULL);
-    ~GuiTick();
     void process();
 
   private:
-    ControlObject* m_pCOGuiTickTime;
-    ControlObject* m_pCOGuiTick50ms;
+    std::unique_ptr<ControlObject> m_pCOGuiTickTime;
+    std::unique_ptr<ControlObject> m_pCOGuiTick20ms;
+    std::unique_ptr<ControlObject> m_pCOGuiTick50ms;
     PerformanceTimer m_cpuTimer;
-    mixxx::Duration m_lastUpdateTime;
+    mixxx::Duration m_lastUpdateTime20ms;
```

Comment:
1 second / 60 frames per second = 16.667 ms. Maybe this frequency should increase a bit.

Reply:
Yea... I don't know what a good choice is. The good thing is that if the user's configured FPS is lower than this, it will match that. This is more of a "max" FPS because of that.

Reply:
1 / 0.02s = 50 frames per second. I don't notice any downside to changing the period to 16.66 ms. Maybe someone with sensitive eyes and a nice screen will notice.

Reply:
Considering we let users configure the waveform framerate up to 60 FPS, I think it would make sense to use the same limit here.

Reply:
the 20 ms might interfere with the default 33,3 ms that drives the timer. 
Instead of a fixed duration we may just pass through the user configure frame rate of GuiTick::process() using m_pCOGuiTickTime

Reply:
I have just tested it and this way the jitter is gone. If this still solved the issue ... perfect.

Reply:
Yea great point -- using guiTickTime itself SGTM.

Reply:
Done.